src/OSPSuite.Core/Domain/Descriptors/NotMatchTagCondition.cs

```diff
@@ -18,6 +18,6 @@ public override bool IsSatisfiedBy(EntityDescriptor entityDescriptor)
          return !entityDescriptor.Tags.Contains(Tag);
       }
 
-      public override IDescriptorCondition CloneCondition() => new NotMatchTagCondition(Tag);
```

Comment:
@Yuri05 @rwmcintosh I have removed this interface. This may break mobi not sure. I'll update and see
This was definitely overkill on domain side

Reply:
ok